arch/risc-v/include/common/irq.h

```diff
@@ -121,7 +121,13 @@
   #define INT_XCPT_REGS     33
 #endif
 
-#define INT_XCPT_SIZE       (8 * INT_XCPT_REGS)
+#ifdef CONFIG_ARCH_RV32
+#define INT_REG_SIZE        4
+#else
+#define INT_REG_SIZE        8
+#endif
```

Comment:
#define INT_REG_SIZE sizeof(uinptr_t)

Reply:
Can't do this since uintptr_t is invalid in assembly.